--- origsrc/twolame-0.3.13/libtwolame/Makefile.am	2013-12-09 17:59:04.745116500 -0600
+++ src/twolame-0.3.13/libtwolame/Makefile.am	2013-12-09 18:21:38.803564200 -0600
@@ -1,4 +1,4 @@
-AM_CFLAGS = -I ../build/ $(WARNING_CFLAGS)
+AM_CFLAGS = -I ../build/ $(WARNING_CFLAGS) -DLIBTWOLAME_DLL_EXPORTS
 
 lib_LTLIBRARIES = libtwolame.la
 include_HEADERS = twolame.h
--- origsrc/twolame-0.3.13/libtwolame/twolame.h	2011-01-01 13:15:01.000000000 -0600
+++ src/twolame-0.3.13/libtwolame/twolame.h	2013-12-09 18:11:54.012116000 -0600
@@ -44,8 +44,12 @@ extern "C" {
 #define DLL_EXPORT
 #else
 #ifdef LIBTWOLAME_DLL_EXPORTS
+#ifdef PIC
 #define DLL_EXPORT __declspec(dllexport)
 #else
+#define DLL_EXPORT
+#endif
+#else
 #define DLL_EXPORT __declspec(dllimport)
 #endif
 #endif
